meetings/2018-02-16_UserFeedbackmeeting.md

```diff
@@ -0,0 +1,189 @@
+* Bradley: Start testing before LTS
+* Tierney: In your production environment how do you handle upgrades,
+           security updates, patches?
+* Joel: we are going with the entire update pull request,
+           QA sign off and then deploy
+* Bradley: pretty much the same
+* Aaron: got recently 8.x, most of the platforms 6.x
+* Nikolay: we have 8 version in production; using TurboFan Ignition boosted our performance
+* Tierney: Is anyone actually using yarn either for dev or production?
+* Joel: Has some issues that our build generally doesn’t work well. 
+  In my testing with our real-word applications from 6 version to 8 version, 
+  we saw a 60% improvement in performance
+* **Survey Q#4:** What is your primary use case for Node?
+  * 	**Answered 294:**
+   	    * back-end API services 43%
+      	* service oriented 3%
+      	* microservice-based 19%
+      	* generating serving dynamic 8%
+      	* SPA applications 9%
+      	* Agents and Data Collectors 2%
+      	* Web Developer Tooling 9%
+      	* Small scripts 1%
+ * Joel:  we would like to see more on this case: 
+ how to use Node direct connect to back -end databases,
+ because now our apps when they need to connect DB they go through service that’s reading Java.  
+ * Bradley: we don’t auto scale our build services, 
+ we have service-oriented architecture with immutable builds
+ 
+* **Survey Q#6:** Do you have infrastructure in place for tracking performance on your applications?
+      	Answered 288: Yes 38%, Work in progress 30%, No 32%
+* Joel: we have a logging to Splunk for general application: 
+less about performance but more about the application health and monitoring 
+usage on our website. APM infrastructure based on Kafka.
+* Bradley: APMs running; need to be turned off in some cases 
+in production due to leaking memory, essentially while they try to track events.
+Reccomendation: include node report in all green applications 
+(get some reports started last month).
+* Nikolay: Splunk for logging, using New Relic as monitoring systems and to be ready for Black Fridays.
+Monitoring systems are big impact on performance
+ 
+**Survey Q#9:** What type of optimization passes do you run on your JavaScript you deploy it to Node.Js?
+ 	Answered 211: Minification 68%, Bundling 60%, Obfuscation 20%, Other 22%
+  	
+**Survey Q#16:** Do you have use cases where you cannot use Node.js because it is single threaded?
+ 	Answered 282: Yes 22%, No 78%
+* Nikolay: anything that need to use cpu that’s a bottleneck
+* Joel: a single threat definitely is something that we’re constantly trying to have to work around it, because we do e-commerce and the server can general do one to two requests per second    
+ 
+## Questions, remarks:
+* Michael: we can help with scripts for IBM I platforms/ running on same benchmark.
+* Tierney: Should we provide a way to go in benchmark for like user land?
+* Michael: Is something that users can contribute to cover up of our Matrix,
```

Comment:
Is something that users can contribute to in order to increase coverage in our matrix.